src/views/AnimalDetails.vue

```diff
@@ -12,7 +12,17 @@
     methods: {
         removeTags(str) {
         return str.replace( /(<([^>]+)>)/ig, '');
-      }
+      },
+      postFavoriteAnimal() {
+        console.log("are you even working?")
+        const favAnimal = {user_id: store.user.id, animal_id: store.animalDetails.id}
+        return fetch('https://secure-island-06435.herokuapp.com/api/v1/users', {
+          method: 'POST',
+          body: JSON.stringify(favAnimal),
+          headers: {'Content-Type': 'application/json'}
+        }).then(response => response.json())
+          .catch(error => console.log(error))
```

Comment:
It would be nice to change console.log to console.warn to see the error as warning in the dev tool. Is there any error handling functionality for this post request?